### DISCUSSION TITLE
Version 8 release!

### DISCUSSION THREAD

**Discussion by tephenavies** [TerriaJS/terriajs#5729] [Announcements]
Today 13 Aug 2021, we're switching over our repos to make version 8 the default you get when you clone terriajs & TerriaMap. Going forwards we will be using the following branches on both repositories (terriajs & TerriaMap):

* `main` - the main stable branch. Now upgraded to terriajs version 8
* `terriajs7` - a stable branch based on terriajs version 7

This will be closely followed by an 8.0.0 npm release (from `main`). We've also just updated our [migration guide](https://docs-v8.terria.io/guide/contributing/migration-guide/) and our [catalog converter](https://catalog-converter.terria.io/) (and [catalog converter CLI](https://github.com/TerriaJS/catalog-converter)) to help you upgrade to version 8!

**Comment by hardreddata** [comment]
Great job.

The new main branch is working fine on Linux and Windows (node 10.15.0).
